recipes/libsoup/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "libsoup" %}
+{% set version = "2.70.0" %}
+{% set version_majmin = '.'.join(version.split('.', 2)[:2]) %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://download.gnome.org/sources/{{ name }}/{{ version_majmin }}/{{ name }}-{{ version }}.tar.xz
+  sha256: 54b020f74aefa438918d8e53cff62e2b1e59efe2de53e06b19a4b07b1f4d5342
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - meson
+    - ninja
+    - pkg-config
+    - gobject-introspection
```

Comment:
gobject-intro is probably a host dep

Reply:
I tried that, but then the gobject-introspection python based command-line utilities didn't work as they picked up the wrong python interpreter. The build and host environments had different versions of python installed and this became quite a mess...